bundle/edu.gemini.ags.servlet/src/main/java/edu/gemini/ags/servlet/ToContext.java

```diff
@@ -183,13 +186,17 @@ public ObsContext apply(HttpServletRequest req) throws RequestException {
             ((PosAngleConstraintAware) inst).setPosAngleConstraint(pac);
         } // otherwise pac is ignored!
 
+        // --- instrument configurable with GeMS?
+        AbstractDataObject aoComp = null;
+        if (inst instanceof Gsaoi) {
+            aoComp = getGeMS(req);
+        }
```

Comment:
If F2 + GeMS is ever commissioned, we will have to figure out how to handle this, as `estimate` always picks the first available strategy in a prioritized list of strategies for an instrument.